services/algolia-sync/src/index.js

```diff
@@ -0,0 +1,23 @@
+const minimist = require('minimist');
+const receiveMessage = require('./components/aws-sqs/sqs');
+const siteSync = require('./components/algolia/site-sync');
+
+const { log } = console;
+
+const args = minimist(process.argv.slice(2), {
+  default: {
+    limit: 500,
+    skip: 0,
+  },
+});
+
+const run = () => {
+  if (args.tenant) {
+    siteSync.setup(args.tenant, args.limit, args.skip);
+  } else {
```

Comment:
When converting to `yargs`, make these two different commands. One to watch the sqs queue, and one to (re)-populate algolia's indices.

Reply:
I updated it use the yargs handler()